firmware/screen/src/main.cpp

```
@@ -0,0 +1,401 @@
+  // Target line (static).
+  char hostbuf[96];
+  snprintf(hostbuf, sizeof(hostbuf), "target  ws://%s:%d%s",
+           SIDECAR_HOST, SIDECAR_PORT, SIDECAR_PATH);
+  draw_row(ROW_HOST, hostbuf, COL_DIM);
+
+  // Demo button (virtual - no touch yet).
+  gfx->fillRoundRect(BTN_X, BTN_Y, BTN_W, BTN_H, 10, COL_BTN_BG);
+  gfx->setTextColor(COL_BTN_FG, COL_BTN_BG);
+  gfx->setTextSize(2);
+  const char* btn_lbl = "auto-ping: toggleFocusPractice";
+  int16_t x1, y1;
```

Clang (14.0.6)

[warning] variable 'x1' is not initialized

(cppcoreguidelines-init-variables)